pip-requirements.txt

```diff
@@ -0,0 +1,2 @@
+mock>=2.0.0
+coveralls
```

Comment:
Also, I think something to bear in mind soon is that this `requirements.txt` covers the bare essentials for running the tests (which is good). But there's not enough here to actually run something mslice like. You might consider very soon (different PR) adding a separate script to pull in `mantid`. @martyngigg what do you think?